packages/web3torrent/src/components/torrent-info/TorrentInfo.tsx

```diff
@@ -13,7 +13,7 @@ const TorrentInfo: React.FC<TorrentInfoProps> = ({torrent}: TorrentInfoProps) =>
         <span className="fileName">{torrent.name}</span>
         <span className="fileSize">{torrent.length}Mb</span>
         {torrent.status ? <span className="fileStatus">{torrent.status}</span> : false}
-        <span className="fileCost">Est. cost ${torrent.cost}</span>
+        <span className="fileCost">Est. cost ${Number(torrent.cost).toFixed(2)}</span>
```

Comment:
Shouldn't `torrent.cost` be typed as `number` to prevent the conversion with the `Number(...)` function?

Reply:
Umm... it's typed as number | undefined, and for now I would like to leave it like that, as that is not a property exposed or even handled by the library, and I don't want to make assumptions of what is the correct behaviour.